build/common.sh

```diff
@@ -407,6 +410,7 @@ function kube::build::clean() {
 # Set up the context directory for the kube-build image and build it.
 function kube::build::build_image() {
   mkdir -p "${LOCAL_OUTPUT_BUILD_CONTEXT}"
+  chown -R ${USER_ID}.${GROUP_ID} ${LOCAL_OUTPUT_BUILD_CONTEXT}
```

Comment:
might be worth adding a comment as to why we're doing this.

also minor style nit - put vars in quotes (e.g. `"${LOCAL_OUTPUT_BUILD_CONTEXT}"`)